### ISSUE TITLE
Changing learning rate

### ISSUE THREAD

**Issue by sobhani** [NVIDIA/pix2pixHD#198]
Do you think if we play with the learning rate to higher (?) or lower(?), might affects the performance of the networks having better-generated results?  else what could help for generating better outcomes?

**Comment by kex243** [comment]
Yes, if to think about traning process as if it is search for optimal function it can have some local minimus which can be achived by increasing LR for some epochs,just like jumping out of fake local minimum. And I do this if traning stucks and is far from result. But usualy decreasing it even more (from 0.002 to 0.0001) than it is being decreased by default helped me to fine tune model in the end.
